### PR TITLE
Added docker support

### LINKED ISSUE #6: Add Docker support
Add a dockerfile to the project, so it can be easily be initialized as a container on Docker.

Consider adding it on Docker-hub (Docker-Store?) and update it automatically upon a successful change.

### PR DESCRIPTION
**Pull request by liadmagen**
Added a docker file and added a command for the Server Manager, enabling setting the IP/port to 0:0:0:0

The port, though should be configured from a setting.

fix #6

### CODE REVIEW

**Comment by Createdd** [comment]
@liadmagen  
How do I start and test this? I have no experience with docker. Maybe you can add instructions to the Readme file as well?

**Comment by liadmagen** [comment]
Regarding docker... I can reference some tutorials. It will be long to describe how to use it, on every OS :) but I can add the command how to use it after it is installed.

**Comment by Createdd** [comment]
@liadmagen 
Alright cool! 
so I would suggest to either add instruction within this PR or create an issue and reference it in this PR. whatever you like more :) 

Also, there is a merge conflict that must be resolved. Do you want to do that or shall I?

File: README.md [hunk 1]
```diff
@@ -71,10 +66,12 @@ conda create -q -n web-miner python=3.6 # create the environment
...
 
 pip install -r requirements.txt # install the packages
-
-python manage.py server # start server
 ```
 
+and test your installation by running the web server:
+```bash
+flask run # start server
+
```
Comment (Createdd) [inline README.md]: you need to add ` ```  ` at the end to close the bash syntax highlighting

File: requirements/dev.txt [hunk 2]
```diff
@@ -3,5 +3,4 @@
 pip
 wheel
 flake8
-sphinx
-flask-script
```
Comment (Createdd) [inline requirements/dev.txt]: please use also 
`pipenv uninstall flask-script`
to uninstall the packages properly from the pipfile as well

Comment (Createdd) [reply requirements/dev.txt]: done

File: README.md (hunk 1 above)
Comment (Createdd) [reply README.md]: done

**Pull request merged**